src/main/java/br/com/metricminer2/scm/GitRepository.java

```diff
@@ -111,8 +119,12 @@ public SCMRepository info() {
 
 	}
 
-	protected Git openRepository() throws IOException {
-		return Git.open(new File(path));
+	protected Git openRepository() throws IOException, GitAPIException {
+		Git git = Git.open(new File(path));
+		if(this.masterBranchName == null) {
```

Comment:
should it be null? or should the default be `master`? How's the behavior of `git.getRepository().getBranch()`? we definitely need an automated test for the case which the master branch is not the main one.